src/main/java/com/cag/cagbackendapi/config/SpringConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.cag.cagbackendapi.config;
+
+import com.cag.cagbackendapi.CagBackendApiApplication;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SpringConfig {
+
+    @Bean
```

Comment:
What does logger do?

Reply:
It just prints out messages like System.out.println("Message"). It's useful if we start tracking and storing our log files which we'll probably be able to do on AWS very easily because then we'll have more clues if something breaks if we logged messages